resources/views/backend/business/show.blade.php

```diff
@@ -0,0 +1,40 @@
+@extends('backend.layouts.master')
+@section('title', trans('labels.business'))
+@section('content')
+    <div class="col-md-12 col-sm-12 col-xs-12 profile_left">
+
+        <div class="profile_img">
+
+            <!-- end of image cropping -->
+            <div id="crop-avatar" class="col-md-2">
+                <!-- Current avatar -->
+                <img class="img-responsive avatar-view" src="http://placehold.it/200x200" alt="Avatar" title="Change the avatar">
```

Comment:
[rank A] do we use this placeholder image for other? I think we may set it as constant for better.